ci-operator/step-registry/openstack/install/multiplenetworks/openstack-install-multiplenetworks-workflow.yaml

```diff
@@ -0,0 +1,35 @@
+workflow:
+  as: openstack-install-multiplenetworks
+  steps:
+    pre:
+    # from chain: ipi-conf-openstack
+    - ref: ipi-conf-openstack
```

Comment:
Why don’t you call the chain?

however, the STEP (ref) ipi-conf-openstack is now:
- openstack-conf-clouds
- openstack-conf-resourcenames

Reply:
^^that's exactly why :) easier to debug if all the steps are listed. 
This is not really going to merge.

Reply:
done